profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/features/VolumeApiLiveTest.java

```diff
@@ -53,13 +55,19 @@ public void teardownTest() {
    @Test
    public void testCreateVolume() {
       assertNotNull(dataCenter);
+      
+      HashSet<String> sshKeys = new HashSet<String>();
+      sshKeys.add("hQGOEJeFL91EG3+l9TtRbWNjzhDVHeLuL3NWee6bekA=");
 
       testVolume = volumeApi().createVolume(
               Volume.Request.creatingBuilder()
               .dataCenterId(dataCenter.id())
               .name("jclouds-volume")
-              .size(3)
+              .size(4)
               .licenceType(LicenceType.LINUX)
+              .type(VolumeType.HDD)
+              .image("362c903a-df61-11e5-80a4-52540005ab80")
```

Comment:
The same way you create the datacenter in the before class, can't you get the list of available images and get a proper one?

Once everything is in place we'll configure our CI system to run live tests weekly. Apart from that, we try to run live tests in every release, to validate everything is OK. Having to do a change at release time if an image id has changed can be kinda frustrating for the releaser :)